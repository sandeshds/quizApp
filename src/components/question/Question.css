.questionBlock {
    margin-left: 5%;
    font-size: inherit;
}

.questionCounter {
    font-size: small;
}

.question {
    font-size: large;
    margin-left: 5px;
}

.optionBlock {
    margin-top: 40px;
    font-size: medium
}

.option {
    margin-top: 10px;
}