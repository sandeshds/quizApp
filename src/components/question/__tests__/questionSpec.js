import React, {Component} from 'react';
import renderer from 'react-test-renderer';
import Question from './../index';

describe('Button', ()=>{
    it('should render question properly', ()=>{
        let questions = [{
            "text": "This is the first question",
            "options": ["answer1","answer2","answer3","answer4"],
            "answer": 2
        }];
        let tree = renderer.create(
            <Question 
                allQuestions = {questions}
                questionToDisplay = {0}
            />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});