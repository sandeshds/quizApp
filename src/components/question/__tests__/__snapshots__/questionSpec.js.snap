// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question should render question properly 1`] = `
<div
  className="questionBlock"
>
  <div>
    <span
      className="questionCounter"
    >
      Javascript Quiz 
      1
       of 
      1
       
    </span>
    <br />
    <span
      className="question"
    >
      This is the first question
    </span>
  </div>
  <div
    className="optionBlock"
  >
    <div
      className="option"
    >
      <div>
        A
         - 
        answer1
      </div>
    </div>
    <div
      className="option"
    >
      <div>
        B
         - 
        answer2
      </div>
    </div>
    <div
      className="option"
    >
      <div>
        C
         - 
        answer3
      </div>
    </div>
    <div
      className="option"
    >
      <div>
        D
         - 
        answer4
      </div>
    </div>
  </div>
</div>
`;
